packer/packer_source.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+ERR_PACKER_COPY_FILE=113 # Error writing a file to disk during VHD CI
+
+copyPackerFiles() {
+  SYSCTL_CONFIG_SRC=/home/packer/sysctl-d-60-CIS.conf
+  SYSCTL_CONFIG_DEST=/etc/sysctl.d/60-CIS.conf
+  RSYSLOG_CONFIG_SRC=/home/packer/rsyslog-d-60-CIS.conf
+  RSYSLOG_CONFIG_DEST=/etc/rsyslog.d/60-CIS.conf
+  ETC_ISSUE_CONFIG_SRC=/home/packer/etc-issue
+  ETC_ISSUE_CONFIG_DEST=/etc/issue
+  ETC_ISSUE_NET_CONFIG_SRC=/home/packer/etc-issue.net
+  ETC_ISSUE_NET_CONFIG_DEST=/etc/issue.net
+  SSHD_CONFIG_SRC=/home/packer/sshd_config
+  SSHD_CONFIG_DEST=/etc/ssh/sshd_config
+  MODPROBE_CIS_SRC=/home/packer/modprobe-CIS.conf
+  MODPROBE_CIS_DEST=/etc/modprobe.d/CIS.conf
+  PWQUALITY_CONF_SRC=/home/packer/pwquality-CIS.conf
+  PWQUALITY_CONF_DEST=/etc/security/pwquality.conf
+  PAM_D_SU_SRC=/home/packer/pam-d-su
+  PAM_D_SU_DEST=/etc/pam.d/su
+  PROFILE_D_CIS_SH_SRC=/home/packer/profile-d-cis.sh
+  PROFILE_D_CIS_SH_DEST=/etc/profile.d/CIS.sh
+  AUDITD_RULES_SRC=/home/packer/auditd-rules
+  AUDITD_RULES_DEST=/etc/audit/rules.d/CIS.rules
+  LABEL_NODES_SRC=/home/packer/label-nodes.sh
+  LABEL_NODES_DEST=/opt/azure/containers/label-nodes.sh
+  LABEL_NODES_SERVICE_SRC=/home/packer/label-nodes.service
+  LABEL_NODES_SERVICE_DEST=/etc/systemd/system/label-nodes.service
+  CIS_SRC=/home/packer/cis.sh
+  CIS_DEST=/opt/azure/containers/provision_cis.sh
+  APT_PREFERENCES_SRC=/home/packer/apt-preferences
+  APT_PREFERENCES_DEST=/etc/apt/preferences
+  KMS_SERVICE_SRC=/home/packer/kms.service
+  KMS_SERVICE_DEST=/etc/systemd/system/kms.service
+  if [[ ${UBUNTU_RELEASE} == "16.04" ]]; then
+    SSHD_CONFIG_SRC=/home/packer/sshd_config_1604
+  fi
+  DIR=$(dirname "$SYSCTL_CONFIG_DEST") && mkdir -p ${DIR} && cp $SYSCTL_CONFIG_SRC $SYSCTL_CONFIG_DEST && chmod 644 $SYSCTL_CONFIG_DEST || exit $ERR_PACKER_COPY_FILE
```

Comment:
we could make this a function to avoid repeating the same line so many times

Reply:
Done